File: Pemprograman_6/Bahasa_C/PRAK602-2510817120005-NorKhalisyah.c
#include <stdio.h>

int main() {
    int n;

    printf("Masukkan jumlah ruangan: ");
    scanf("%d", &n);

    int zetsu[1000];

    printf("Masukkan jumlah zetsu di setiap ruangan: ");
    for(int i = 0; i < n; i++) {
        scanf("%d", &zetsu[i]);
    }

    printf("Hasil setelah membelah diri:\n");
    for(int i = 0; i < n; i++) {
        printf("%d ", zetsu[i] * (i + 1));
    }

    return 0;
}

File: Pemprograman_6/Bahasa_Python/PRAK602-2510817120005-NorKhalisyah.py
n = int(input("Masukkan jumlah ruangan: "))

zetsu = list(map(int, input("Masukkan jumlah zetsu di setiap ruangan: ").split()))

hasil = []
for i in range(n):
    hasil.append(zetsu[i] * (i + 1))

print("Hasil setelah membelah diri:")
for h in hasil:
    print(h, end=" ")
